src/component/handlers/edit/DraftEditorEditAndroidHandler.js

```diff
@@ -61,6 +62,7 @@ function handleKeyDown(editor, e) {
 // Only handle the following events:
 const DraftEditorEditAndroidHandler = {
   onBeforeInput,
+  onBlur,
```

Comment:
⭐️⭐️⭐️: Do we need a parallel on focus that pulls in `editOnFocus`?

Reply:
Good point. Added!